orc8r/cloud/go/services/configurator/storage/storage.proto

```diff
@@ -208,11 +208,20 @@ message EntityLoadCriteria {
     bool load_assocs_from_this = 4;
 
     bool load_permissions = 5;
+
+    // The following parameters allow for pagination of entity loads.
+    // Size of page to return
+    uint64 page_size = 6;
```

Comment:
I think uint32 is sufficient. Consider a max uint32 page size retrieving 1-byte objects -- that's a 4gb page. And more realistic is like, 30-byte objects => 130gb. I don't anticipate us running into this requirement